20220208 release (#1108)
publiccodenet/about: organization/governance-model.md

These decisions are agreed through a majority vote. Voting rights are proportional to members' membership dues. This ensures democratic ownership and control by all members.

> The general assembly meets at least once every year. Minutes from the general assembly meetings are shared with all members and [publicly archived](/general-assemblies/index.md)

Reviewer: ```suggestion
The general assembly meets at least once every year. Minutes from the general assembly meetings are shared with all members and [publicly archived](/general-assemblies/index.md).
```

### Advisory board

> The advisory board is a set of experts that support the general assembly, board of directors and staff. There are some decisions for which they must be consulted.

Reviewer: ```suggestion
The advisory board members are experts that support the general assembly, board of directors and staff. There are some decisions for which they must be consulted.
```
